microsite/data/plugins/pulumi.yaml

```diff
@@ -0,0 +1,10 @@
+---
+title: Pulumi
+author: pulumi.com
```

Comment:
Totally fine with `pulumi.com`, but it could also be just "Pulumi" again as it is more common to use the brand name currently :) 

<img width="879" alt="Screenshot 2023-09-27 at 13 29 48" src="https://github.com/backstage/backstage/assets/8904624/c8717b6f-72ce-4500-b8c0-a98d54545d2e">

```suggestion
author: Pulumi
```